addons/mail/static/src/discuss/core/common/thread_model_patch.js

```diff
@@ -27,15 +27,13 @@ const threadPatch = {
             },
         });
         this.offlineMembers = Record.many("ChannelMember", {
-            /** @this {import("models").Thread} */
-            compute() {
-                return this.channelMembers.filter(
-                    (member) => member.persona?.im_status !== "online"
-                );
-            },
+            compute: this._computeOfflineMembers,
             sort: (m1, m2) => (m1.persona?.name < m2.persona?.name ? -1 : 1),
         });
     },
+    _computeOfflineMembers() {
+        return this.channelMembers.filter((member) => member.persona?.im_status !== "online");
+    },
```

Comment:
This was already there before PR, so not a huge deal, but not that compute is not really correct: members can have an im_status of `leave_online` / `away` / `bot`, each of which should be actually considered as "online". Replacing by `.includes("offline")` solves this issue.